### PR TITLE
chore: update sfyra dependecies

### PR DESCRIPTION
**Pull request by AlekSi**
Sidero:

```diff
-github.com/envoyproxy/go-control-plane v0.9.9-0.20201210154907-fd9021fe5dad
+github.com/envoyproxy/go-control-plane v0.9.9-0.20210217033140-668b12f5399d
-github.com/go-logr/logr v0.2.1-0.20200730175230-ee2de8da5be6
+github.com/go-logr/logr v0.4.0
+github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
-github.com/golang/protobuf v1.4.3
+github.com/golang/protobuf v1.5.0
-github.com/google/go-cmp v0.5.4
+github.com/google/go-cmp v0.5.5
-github.com/hashicorp/go-multierror v1.1.0
+github.com/hashicorp/go-multierror v1.1.1
-github.com/nxadm/tail v1.4.4
+github.com/nxadm/tail v1.4.8
-github.com/onsi/ginkgo v1.15.0
-github.com/onsi/gomega v1.10.1
+github.com/onsi/ginkgo v1.16.1
+github.com/onsi/gomega v1.11.0
-github.com/talos-systems/go-procfs v0.0.0-20201223150035-a82654edcec1
+github.com/talos-systems/go-procfs v0.0.0-20210108152626-8cbc42d3dc24
-github.com/talos-systems/talos/pkg/machinery v0.0.0-20210401163915-1d8e9674a91b
+github.com/talos-systems/talos/pkg/machinery v0.0.0-20210416105550-2b83440d6f7f
-go.uber.org/zap v1.14.1
+go.uber.org/zap v1.16.0
-golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
-golang.org/x/sys v0.0.0-20210112080510-489259a85091
+golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+golang.org/x/sys v0.0.0-20210426230700-d19ff857e887
-google.golang.org/grpc v1.36.0
-google.golang.org/protobuf v1.25.0
+google.golang.org/grpc v1.37.0
+google.golang.org/protobuf v1.26.0
-gopkg.in/yaml.v2 v2.3.0
+gopkg.in/yaml.v2 v2.4.0
```

Sfyra:
```diff
+github.com/davecgh/go-xdr v0.0.0-20161123171359-e6a2ba005892
-github.com/envoyproxy/go-control-plane v0.9.9-0.20201210154907-fd9021fe5dad
+github.com/envoyproxy/go-control-plane v0.9.9-0.20210217033140-668b12f5399d
-github.com/go-logr/logr v0.2.1
+github.com/go-logr/logr v0.4.0
+github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0
-github.com/golang/protobuf v1.4.3
+github.com/golang/protobuf v1.5.0
-github.com/google/go-cmp v0.5.4
+github.com/google/go-cmp v0.5.5
-github.com/hashicorp/go-multierror v1.1.0
+github.com/hashicorp/go-multierror v1.1.1
-github.com/nxadm/tail v1.4.4
+github.com/nxadm/tail v1.4.8
-github.com/onsi/ginkgo v1.15.0
-github.com/onsi/gomega v1.10.3
+github.com/onsi/ginkgo v1.16.1
+github.com/onsi/gomega v1.11.0
-github.com/talos-systems/go-loadbalancer v0.1.1-0.20201015151439-a4457024d518
+github.com/talos-systems/go-loadbalancer v0.1.1
-github.com/talos-systems/sidero v0.1.0-alpha.1.0.20200915181156-11a0a80e3d8b => ../
-github.com/talos-systems/talos v0.9.1
-github.com/talos-systems/talos/pkg/machinery v0.0.0-20210401163915-1d8e9674a91b
+github.com/talos-systems/sidero v0.0.0-00010101000000-000000000000 => ../
+github.com/talos-systems/talos v0.9.3
+github.com/talos-systems/talos/pkg/machinery v0.0.0-20210416105550-2b83440d6f7f
+github.com/vmware/govmomi v0.24.0
-golang.org/x/sys v0.0.0-20210301091718-77cc2087c03b
+golang.org/x/sys v0.0.0-20210426230700-d19ff857e887
-google.golang.org/grpc v1.36.0
-google.golang.org/protobuf v1.25.0
+google.golang.org/grpc v1.37.0
+google.golang.org/protobuf v1.26.0
```

### CODE REVIEW

**Review by smira — DISMISSED** [review]

**Comment by AlekSi** [comment]
I doesn't look possible to pull pkg/machinery 0.10 without Talos 0.10

File: sfyra/go.mod [hunk 1]
```diff
@@ -2,24 +2,29 @@ module github.com/talos-systems/sidero/sfyra
...
+	github.com/talos-systems/go-loadbalancer v0.1.1
 	github.com/talos-systems/go-procfs v0.0.0-20210108152626-8cbc42d3dc24
 	github.com/talos-systems/go-retry v0.2.1-0.20210119124456-b9dc1a990133
 	github.com/talos-systems/net v0.2.1-0.20210212213224-05190541b0fa
-	github.com/talos-systems/sidero v0.1.0-alpha.1.0.20200915181156-11a0a80e3d8b
-	github.com/talos-systems/talos v0.9.1
-	github.com/talos-systems/talos/pkg/machinery v0.0.0-20210401163915-1d8e9674a91b
-	google.golang.org/grpc v1.36.0
+	github.com/talos-systems/sidero v0.0.0-00010101000000-000000000000
+	github.com/talos-systems/talos v0.9.3
```
Comment (smira) [inline sfyra/go.mod:25]: if we pull new Talos, it makes sense to update `TALOS_RELEASE` in the main `Makefile` as well to update what we're testing with

**Review by smira — APPROVED** [review]

**Review by smira — APPROVED** [review]

**Pull request merged**